client/signup/steps/woocommerce-install/select-site/index.tsx

```diff
@@ -43,31 +42,38 @@ export default function SelectSite( props: WooCommerceInstallProps ): ReactEleme
 		);
 	};
 
-	const [ selectedSiteId, setSelectedSiteId ] = useState();
+	const [ selectedId, setSelectedId ] = useState();
 
+	const siteSlug = useSelector( ( state ) => getSiteSlug( state, selectedId ) );
 	const dispatch = useDispatch();
 
 	useEffect( () => {
-		if ( ! selectedSiteId ) {
+		if ( ! siteSlug || ! selectedId ) {
 			return;
 		}
-		dispatch( submitSignupStep( { stepName: 'select-site' }, { siteId: selectedSiteId } ) );
+
+		dispatch( submitSignupStep( { stepName: 'select-site' }, { site: siteSlug } ) );
 		goToStep( 'confirm' );
-	}, [ selectedSiteId, goToStep, dispatch ] );
+	}, [ siteSlug, selectedId, goToStep, dispatch ] );
 
+	const title = __( 'Select a site' );
+	const subtitle = __( 'Pick which site you would like to use to set up your new store.' );
 	return (
 		<StepWrapper
 			flowName="woocommerce-install"
 			hideSkip={ true }
 			nextLabelText={ __( 'Confirm' ) }
-			allowBackFirstStep={ true }
-			backUrl="/woocommerce-installation"
+			shouldHideNavButtons={ true }
 			className="select-site__step-wrapper"
 			align={ isReskinned ? 'left' : 'center' }
 			isWideLayout={ isReskinned }
+			headerText={ title }
+			fallbackHeaderText={ title }
+			subHeaderText={ subtitle }
+			fallbackSubHeaderText={ subtitle }
```

Comment:
I'm not sure, but I feel like these were removed to fix something related to the confirm screen. It all seems to be working at the moment though.

Reply:
Suspect similar change needs to happen in the confirm screen, (moving them out of the flow file and into the step itself) but its not currently causing an error in any case could find.